### PR TITLE
feat: [PIE-6718]: pipeline services generated for git caching

### PR DESCRIPTION
**Pull request by bhavya-sinha**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [x] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [x] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by devinder-harness — APPROVED** [review]

**Review by AshwinBhatkal-Harness — APPROVED** [review]

**Comment by bhavya-sinha** [comment]
retrigger cypress-rest

File: src/services/cv/index.tsx [hunk 1]
```diff
@@ -503,20 +503,20 @@ export type CEAwsConnector = ConnectorConfigDTO & {
   awsAccountId?: string
   crossAccountAccess: CrossAccountAccess
   curAttributes?: AwsCurAttributes
-  featuresEnabled?: ('BILLING' | 'OPTIMIZATION' | 'VISIBILITY' | 'GOVERNANCE')[]
+  featuresEnabled?: ('BILLING' | 'OPTIMIZATION' | 'VISIBILITY' | 'GOVERNANCE' | 'COMMITMENT_ORCHESTRATOR')[]
```
Comment (deepesh-ui) [inline src/services/cv/index.tsx:506]: Shouldn't we be updating swagger file as well ?

Comment (harshilbits) [inline src/services/cv/index.tsx:506]: Yeah this should be automatically generated.

Comment (bhavya-sinha) [reply src/services/cv/index.tsx:506]: This is automatically generated.

Comment (bhavya-sinha) [reply src/services/cv/index.tsx:506]: We dont do it for any other services. We fetch from the url by default. Why do we need swagger for CV?

**Review by harshilbits — APPROVED** [review]

**Review by vkbansal-harness — APPROVED** [review]

**Pull request merged**